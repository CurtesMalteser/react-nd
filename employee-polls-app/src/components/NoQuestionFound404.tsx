
import Container from 'react-bootstrap/esm/Container';
import Col from 'react-bootstrap/esm/Col';
import Row from 'react-bootstrap/esm/Row';
import Image from 'react-bootstrap/esm/Image';
import Logo from '../assets/img/employees_pool_logo.jpg'

function NoQuestionFound404() {

    return (
        <Container className="md-6" style={{ marginTop: 20, marginBottom: 20, marginLeft: "auto", marginRight: "auto" }} >
            <Col>
                <Row className='d-flex justify-content-center'>
                    <Image
                        src={Logo}
                        style={{
                            width: '400px',
                            height: '400px',
                            marginBottom: '48px',
                        }}
                        roundedCircle
                        alt="logo"
                    />
                </Row>
                <Row style={{ marginBottom: "48px" }}>
                    <h1 className="d-flex justify-content-center">Oops! Somenthing went wrong...</h1>
                </Row>
                <Row style={{ marginBottom: "48px" }}>
                    <h2 className="d-flex justify-content-center">404: Question not found</h2>
                </Row>
            </Col>
        </Container>
    )
}

export default NoQuestionFound404;