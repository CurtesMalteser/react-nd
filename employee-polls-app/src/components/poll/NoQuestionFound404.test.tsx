import { render } from '@testing-library/react';
import NoQuestionFound404 from './NoQuestionFound404'; // Adjust the import path as necessary

describe('NoQuestionFound404', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<NoQuestionFound404 />);
    expect(asFragment()).toMatchSnapshot();
  });
});