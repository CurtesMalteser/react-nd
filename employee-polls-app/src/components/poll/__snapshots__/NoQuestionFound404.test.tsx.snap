// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoQuestionFound404 should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="md-6 container"
    style="margin: 20px auto 20px auto;"
  >
    <div
      class="col"
    >
      <div
        class="d-flex justify-content-center row"
      >
        <img
          alt="logo"
          class="rounded-circle"
          src="employees_pool_logo.jpg"
          style="width: 400px; height: 400px; margin-bottom: 48px;"
        />
      </div>
      <div
        class="row"
        style="margin-bottom: 48px;"
      >
        <h1
          class="d-flex justify-content-center"
        >
          Oops! Somenthing went wrong...
        </h1>
      </div>
      <div
        class="row"
        style="margin-bottom: 48px;"
      >
        <h2
          class="d-flex justify-content-center"
        >
          404: Question not found
        </h2>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
